pkg/model/config/config.go

```diff
@@ -119,6 +119,17 @@ func (c Config) HasGroup(group string) bool {
 	return false
 }
 
+// HasWebhook returns true if webhook is already present
+func (c Config) HasWebhook(kind string) bool {
+	for _, r := range c.Resources {
+		if r.Kind == kind {
+			return r.WebhookVersion != ""
```

Comment:
We need to rebase it with masyter change it to `r.Webhook != nil && r.Webhook.WebhookVersion != ""`. More; https://github.com/kubernetes-sigs/kubebuilder/pull/1899